internal/rbd/controllerserver_test.go

```diff
@@ -67,3 +68,149 @@ func TestIsThickProvisionRequest(t *testing.T) {
 		t.Errorf("request should be for thick-provisioning: %s", req.Parameters["thickProvision"])
 	}
 }
+
+func TestIsMigrationVolHandle(t *testing.T) {
+	t.Parallel()
+	volID := "wrong_volume_handle"
+	req := &csi.DeleteVolumeRequest{
+		VolumeId: volID,
+	}
+
+	if isMigrationVolHandle(req.VolumeId) {
+		t.Error("request is not a migration delete request")
+	}
+
+	req.VolumeId = "migrate-alpha_mons-10.70.53.126:6789_kubernetes-dynamic-pvc-e0b45b52-7e09-47d3-8f1b-806995fa4412_pool_replica_pool" // nolint:lll // migration handle
+	if !isMigrationVolHandle(req.VolumeId) {
+		t.Error("request is a migration delete request")
+	}
+
+	req.VolumeId = "migrate-alpha_kubernetes-dynamic-pvc-e0b45b52-7e09-47d3-8f1b-806995fa4412_pool_replica_pool"
+	if !isMigrationVolHandle(req.VolumeId) {
+		t.Error("request is a migration delete request")
+	}
+
+	req.VolumeId = "migrate-alpha_kubernetes-dynamic-pvc-e0b45b52-7e09-47d3-8f1b-806995fa4412"
+	if !isMigrationVolHandle(req.VolumeId) {
+		t.Error("request is a migration delete request")
+	}
+	req.VolumeId = "migrate-alpha"
+	if isMigrationVolHandle(req.VolumeId) {
+		t.Error("request is not a migration delete request")
+	}
+
+	req.VolumeId = "kubernetes-dynamic-pvc-e0b45b52-7e09-47d3-8f1b-806995fa4412"
+	if isMigrationVolHandle(req.VolumeId) {
+		t.Error("request is not a migration delete request")
+	}
+}
+
+func TestParseMigrationVolHandle(t *testing.T) {
```

Comment:
this unittesting should be done at migration_test.go?

Reply:
yep, it can be moved to .